samples/sample/KermitSampleIOS/KermitSampleIOS/ContentView.swift

```diff
@@ -14,13 +14,16 @@ import SwiftUI
 import shared
 
 struct ContentView: View {
-    
+
     let kermit = (UIApplication.shared.delegate as! AppDelegate).kermit
     let common: SampleCommon
-    
+
     init() {
         self.common = SampleCommon(logger: kermit)
         self.kermit.i(message: {"loaded"})
+        LoggerKt.v(message: {""})
```

Comment:
temporary wrt #158, should probably still have a sample that exports to show using Logger from swift